src/med_bench/mediation.py

```diff
@@ -613,177 +452,23 @@ def mediation_multiply_robust(
-        )
-        mu_t0[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t0, m)[test_index, :]
-        )
-
-        # predict E[Y|T=t,M=m,X]
-        mu_t0_m0[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t0, m0)[test_index, :]
-        )
-        mu_t0_m1[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t0, m1)[test_index, :]
-        )
-        mu_t1_m1[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t1, m1)[test_index, :]
-        )
-        mu_t1_m0[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t1, m0)[test_index, :]
-        )
-
-        # E[E[Y|T=1,M=m,X]|T=t,X] model fitting
-        reg_y_t1m1_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t1_m1[test_index][ind_t0]
-        )
-        reg_y_t1m0_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t1_m0[test_index][ind_t0]
-        )
-        reg_y_t1m1_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t1_m1[test_index][~ind_t0]
-        )
-        reg_y_t1m0_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t1_m0[test_index][~ind_t0]
-        )
-
-        # predict E[E[Y|T=1,M=m,X]|T=t,X]
-        E_mu_t1_t0[test_index] = (
-            reg_y_t1m0_t0.predict(x[test_index, :]) * f_00x[test_index]
-            + reg_y_t1m1_t0.predict(x[test_index, :]) * f_01x[test_index]
-        )
-        E_mu_t1_t1[test_index] = (
-            reg_y_t1m0_t1.predict(x[test_index, :]) * f_10x[test_index]
-            + reg_y_t1m1_t1.predict(x[test_index, :]) * f_11x[test_index]
-        )
-
-        # E[E[Y|T=0,M=m,X]|T=t,X] model fitting
-        reg_y_t0m1_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t0_m1[test_index][ind_t0]
-        )
-        reg_y_t0m0_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t0_m0[test_index][ind_t0]
-        )
-        reg_y_t0m1_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t0_m1[test_index][~ind_t0]
-        )
-        reg_y_t0m0_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t0_m0[test_index][~ind_t0]
-        )
-
-        # predict E[E[Y|T=0,M=m,X]|T=t,X]
-        E_mu_t0_t0[test_index] = (
-            reg_y_t0m0_t0.predict(x[test_index, :]) * f_00x[test_index]
-            + reg_y_t0m1_t0.predict(x[test_index, :]) * f_01x[test_index]
-        )
-        E_mu_t0_t1[test_index] = (
-            reg_y_t0m0_t1.predict(x[test_index, :]) * f_10x[test_index]
-            + reg_y_t0m1_t1.predict(x[test_index, :]) * f_11x[test_index]
-        )
+    classifier_y, cross_y_clf, classifier_m, classifier_x = _get_y_m_x_classifiers(regularization, forest, calibration,
```

Comment:
y classifier here too

Reply:
This has been addressed in 3f1b34577643b37b886202ab5873e7b5a9054a40